user_guide_src/source/installation/upgrade_security/002.php

```diff
@@ -6,6 +6,7 @@
 );
 
 // ...
```

Comment:
It seems `// ...` is not needed.
```suggestion
```
![Screenshot 2022-02-25 19 07 35](https://user-images.githubusercontent.com/87955/155696425-b17743a2-888c-464a-95ee-aaa9efb402ad.png)